External/SPMmodified/toolbox/cat12/cat_main.m

```diff
@@ -222,14 +222,14 @@
     end
     if job.extopts.new_release && 0 
       % improvements for large ventricles ... not working now (RD201911)
-      if isfield(res,'Ylesion') && sum(res.Ylesion(:)>0)
+      if isfield(res,'YlesionFull') && sum(res.YlesionFull(:)>0)
```

Comment:
Should we update /ExploreASL/External/README_SPMmodified.txt accordingly for the two CAT12 changes?